cluster/terraform_kubernetes/config/test.tfvars.json

```diff
@@ -51,9 +51,18 @@
     "bat-staging/apply-staging": {
       "receiver": "SLACK_WEBHOOK_ATT",
       "max_cpu": 0.8
+    },
+    "bat-staging/register-staging": {
+      "receiver": "SLACK_WEBHOOK_RTT",
+      "max_cpu": 0.8
```

Comment:
if I run deploy-plan, the output alert rules appear to ignore max_cpu: 0.8, and use the default 0.6

Reply:
updated by adding max_mem value in tfvars file